GNUmakefile

```diff
@@ -206,6 +206,10 @@ check: ## Lint the source code
 	@$(MAKE) proto
 	@if (git status | grep -q .pb.go); then echo the following proto files are out of sync; git status |grep .pb.go; exit 1; fi
 
+	@echo "==> Check format of jobspecs and HCL files..."
+	@$(MAKE) hclfmt
+	@if (git status -s | grep -q '\.hcl\|\.nomad'); then echo the following HCL files are out of sync; git status | grep '\.hcl\|\.nomad'; exit 1; fi
```

Comment:
@notnoop how's this look? I ended up using `git status -s` to just get the files because otherwise the whole output invariably seemed to include `nomad` and I was getting false positives 😀

Reply:
Good point.. You can also do a `git diff --name-only` to print out only the file changes.  We probably could do that with the pb.go check.

Also, I find using `-e` a tiny bit more readable, e.g. `grep -e '\.hcl$' -e '\.nomad$'`?